src/main/java/ch/admin/bag/covidcertificate/web/controller/CovidCertificatePdfGenerateController.java

```diff
@@ -25,7 +25,7 @@
 @Slf4j
 public class CovidCertificatePdfGenerateController {
```

Comment:
Same as with the service: to me a class name containing fromExisting would be more accurate.